### PR TITLE
Create posts folder

### LINKED ISSUE #17: Commands fail with opaque error when _posts does not exist
Running a `jekyll publish`, or `jekyll post` when _posts does not exist results in errors looking like this:
`jekyll 2.5.3 | Error:  No such file or directory @ sys_fail2 - (_drafts/my-site.md, _posts/2015-06-07-my-site.md)`
I think it should create the folder, or at least display an informative error instead.

### PR DESCRIPTION
**Pull request by FellowMD**
Closes #17

### CODE REVIEW

**Comment by parkr** [comment]
Great!

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ end
 
 1. Fork it ( http://github.com/jekyll/jekyll-compose/fork )
 2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Push to the branch (`git push origin my-new-feature`)
-5. Create new Pull Request
+3. Run the specs (`script/test`)
```
Comment (parkr) [inline README.md]: This should probably be `script/cibuild`.

Comment (FellowMD) [inline README.md]: I figured test was for devs and cibuild was for travis. Seems fine to leave those separate to me, but you want me to change it?

Comment (parkr) [inline README.md]: Ultimately we want to make sure that Travis will run, so if Travis runs `script/cibuild`, you should too! :)

**Comment by parkr** [comment]
:shipit: with that one modification above ^

**Pull request merged**